core/metrics-core-service/src/main/java/org/hawkular/metrics/core/dropwizard/HawkularMetricsRegistryListener.java

```diff
@@ -154,4 +275,15 @@ private void createMetrics(String metricName, List<Metric<?>> metrics) {
                 );
     }
 
+    private void deleteMetrics(String metricName, List<MetricId<?>> metrics) {
```

Comment:
> what's the point of metricName here?

Yes it is just for logging. It is the same for each of the constituent metrics whereas the metric id is of course unique for each one.

I agree about the refactoring, but I am going to hold off for the moment. I am going to introduce another ticket that will involve some changes to this code including the remove of some of it. It makes more sense to do the refactoring there.